test/acceptance/tests/terminating-gateway/terminating_gateway_test.go

```diff
@@ -0,0 +1,94 @@
+package connect
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Test that terminating gateways work in a default installation.
+func TestTerminatingGateway(t *testing.T) {
+	testName := "Terminating GW non-acl"
+	t.Run(testName, func(t *testing.T) {
```

Comment:
Are you running this in a `t.Run` to make way for the secure permutation?

Reply:
Yup!  That was the intention, it gets a bit ugly because the workflow for secure is very different so maybe it warrants a second test instead?

Reply:
Yeah not sure, I think if it might not be in this same flow then you could leave this out.

Reply:
I agree it's probably best to leave it out if it's not required for this test.